### PR TITLE
 Fix worker overview page cross platform responsive issues

### LINKED ISSUE #1068: Status Dashboard: Responsive error at MAC
**Description:**
Responsive error at MAC

**Suggested Labels:**
Bug

**Affected Product Version:**
RC2

### PR DESCRIPTION
**Pull request by erangatl**
## Purpose
This is to fix worker overview page cross platform responsive issues.
Fix #1068 , #1211

image

### CODE REVIEW

File: components/org.wso2.carbon.status.dashboard.web/package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "dagre-d3": "^0.6.1",
     "html-loader": "^0.5.0",
     "material-ui": "^0.19.4",
```
[the remark this answers is not in the document]
Comment (erangatl) [reply components/org.wso2.carbon.status.dashboard.web/package.json]: Fully moving to material-ui-next is time consuming due to many API changes, So for the moment we decided to go with both and migrate gradually.

File: components/org.wso2.carbon.status.dashboard.web/src/overview/WorkerOverview.jsx [hunk 2]
```diff
@@ -20,37 +20,35 @@
...
+import {Card, Divider, Button} from "material-ui-next";
 import ContentAdd from "material-ui/svg-icons/content/add";
 //App Components
 import WorkerThumbnail from "./WorkerThumbnail";
 import StatusDashboardAPIS from "../utils/apis/StatusDashboardAPIs";
 import Header from "../common/Header";
 import AuthenticationAPI from "../utils/apis/AuthenticationAPI";
 import AuthManager from "../auth/utils/AuthManager";
-import {FormattedMessage} from "react-intl";
+// import {FormattedMessage} from "react-intl";
```
[the remark this answers is not in the document]
Comment (erangatl) [reply components/org.wso2.carbon.status.dashboard.web/src/overview/WorkerOverview.jsx]: Fixed

File: components/org.wso2.carbon.status.dashboard.web/src/overview/WorkerOverview.jsx [hunk 3]
```diff
@@ -20,37 +20,35 @@
 import React from "react";
 import {Link} from "react-router-dom";
 //Material UI
-import {GridList} from "material-ui/GridList";
+import {GridList, Toggle} from "material-ui";
```
[the remark this answers is not in the document]
Comment (erangatl) [reply components/org.wso2.carbon.status.dashboard.web/src/overview/WorkerOverview.jsx]: Migrated only the required to minimize the issues.

File: components/org.wso2.carbon.status.dashboard.web/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (this) [reply components/org.wso2.carbon.status.dashboard.web/package.json]: ok, I assume both works together correctly.

**Pull request merged**